recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_1.22.0.imx.bb

```diff
@@ -3,7 +3,7 @@
 # recipe. The second section customizes the recipe for i.MX.
 
 ########### OE-core copy ##################
-# Upstream hash: a21649109374fde44cf77de845cfb3cb6cbfb138
+# Upstream hash: fb2d28e0315ece6180c87c7047587673024a09f7
 
 require recipes-multimedia/gstreamer/gstreamer1.0-plugins-common.inc
 require recipes-multimedia/gstreamer/gstreamer1.0-plugins-license.inc
```

Comment:
This should use the local copy.

Reply:
Could you please provide me with more details?

Reply:
The original commit that copied the downgraded recipes didn't copy the .inc files. I guess if we're taking a snapshot we should copy the .inc files as well, right?

Reply:
Sure, it makes sense. Thanks for looking at this.

Reply:
yeah using the inc files available on oe-core will get us out of sync quickly. however since the names are the same I need to confirm locally which inc are used, to check if we use the ones on the layer or the upstream.

Reply:
I'm not sure if you are suggesting that we should keep the full path. In case you are, to be clear, these lines were changed to the full path after copying the recipes in order to find the `.inc` files in oe-core. Changing the lines back just re-aligns them with what is in oe-core.